de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/session/internal/ActivityQueuer.java

```diff
@@ -175,11 +180,11 @@ private void queueActivities(final List<IActivity> activitiesToExecute,
     }
 
     private void flushQueues(final List<IActivity> activities) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Refactor this method to reduce its Cognitive Complexity from 16 to the 15 allowed. ([squid:S3776](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS3776))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=260&issues=AWbfHSW7O08i52vdMzpp&open=AWbfHSW7O08i52vdMzpp)

[ISSUE_KEY]: AWbfHSW7O08i52vdMzpp